docs/getting-started/offline-endpoint.asciidoc

```diff
@@ -0,0 +1,180 @@
+
+* If your endpoints cannot access the internet directly, set up a local HTTP mirror server. Refer to <<artifact-mirror>>.
+* If your endpoints are running in an air-gapped environment, set up a local HTTP server and manually copy global artifact updates. Refer to <<air-gapped-artifact-server>>.
+
+[discrete]
+[[artifact-mirror]]
+== Host an {elastic-endpoint} artifact mirror
+
+You can deploy your own {elastic-endpoint} global artifact mirror to enable endpoints to update their global artifacts automatically through another server acting as a proxy. This allows endpoints to get updates even when they can't directly access the internet. 
+
+Complete these steps:
+
+. Deploy an HTTP reverse proxy server.
+. Configure {elastic-endpoint} to read from the proxy server.
+
+[discrete]
+=== Step 1: Deploy an HTTP reverse proxy server
+
+Set up and configure an HTTP reverse proxy to forward requests to https://artifacts.security.elastic.co and include response headers from the elastic.co server when proxying.
+
+[discrete]
+._Example: Nginx_
+[%collapsible]
+====
+This example script starts an Nginx Docker image and configures it to proxy artifacts:
+
+[source,sh]
+----
+cat > nginx.conf << EOF
+server {
+  location / {
+    proxy_pass https://artifacts.security.elastic.co;
+  }
+}
+EOF
+docker run -v "$PWD"/nginx.conf:/etc/nginx/conf.d/default.conf:ro -p 80:80 nginx
+----
+
+IMPORTANT: This example script is not appropriate for production environments. We recommend configuring the Nginx server to use http://nginx.org/en/docs/http/configuring_https_servers.html[TLS] according to your IT policies. Refer to https://docs.nginx.com/nginx/admin-guide/installing-nginx/[Nginx documentation] for more information on downloading and configuring Nginx. 
+====
+
+[discrete]
+._Example: Apache HTTPD_
+[%collapsible]
+====
+This example script starts an Apache httpd Docker image and configures it to proxy artifacts:
+
+[source,sh]
+----
+docker run --rm httpd cat /usr/local/apache2/conf/httpd.conf > httpd.conf
+cat >> httpd.conf << EOF
+LoadModule proxy_module modules/mod_proxy.so
+LoadModule proxy_http_module modules/mod_proxy_http.so
+LoadModule ssl_module modules/mod_ssl.so
+
+SSLProxyEngine on
+ServerName localhost
+ProxyPass / https://artifacts.security.elastic.co/
+ProxyPassReverse / https://artifacts.security.elastic.co/
+EOF
+docker run -p 80:80 -v "$PWD"/httpd.conf:/usr/local/apache2/conf/httpd.conf httpd
+----
+
+IMPORTANT: This example script is not appropriate for production environments. We recommend configuring httpd to use https://httpd.apache.org/docs/trunk/ssl/ssl_howto.html[TLS] according to your IT policies. Refer to https://httpd.apache.org[Apache documentation] for more information on downloading and configuring Apache httpd.
```

Comment:
```suggestion
IMPORTANT: This example script is not appropriate for production environments. We recommend configuring `httpd` to use https://httpd.apache.org/docs/trunk/ssl/ssl_howto.html[TLS] according to your IT policies. Refer to https://httpd.apache.org[Apache documentation] for more information on downloading and configuring Apache httpd.
```

Reply:
I think plain text is appropriate here, as httpd is the name of the software, not a command or piece of code. Apache's docs also use the name without formatting.